pkg/crds/yaml/generated/management.cattle.io_roletemplates.yaml

```diff
@@ -49,7 +49,7 @@ spec:
             - ""
             type: string
           description:
-            description: Description holds any text desired to better describes the
+            description: Description holds any text desired to better describe the
```

Comment:
```suggestion
            description: Description holds text that describes the resource
```

Reply:
I'm happy to change other fields, but not on generated files like this. Please take a look at the source file, and you can leave a comment there.